examples/visual-tests/amp-story/amp-story-landscape-templates.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  'shows half-half template with image on the left': async (page, name) => {
+    await page.waitFor(1600);
```

Comment:
Do we need to use `waitFor(ms)` here? Does it flake if we just wait for a css class?

Reply:
I updated it, let's see if that works :)

Reply:
For future reference, you can run the tests locally repeatedly to test for these flakes:

```
for i in {1..5}; do gulp visual-diff --nobuild --percy_project ampproject/<<your-username>> --percy_token <<your-percy-token>> --grep "<<your-test--names-regex>>"; done
```